search-commons/src/main/java/no/unit/nva/search/RestHighLevelClientWrapper.java

```diff
@@ -67,4 +69,9 @@ public UpdateResponse update(UpdateRequest updateRequest, RequestOptions request
     public IndicesClientWrapper indices() {
         return new IndicesClientWrapper(client.indices());
     }
+
+    @JacocoGenerated
+    public BulkResponse bulk(BulkRequest request, RequestOptions requestOption) throws IOException {
```

Comment:
this class exists only to make the ElasticSearch client mockable. All methods in this class just call the respective EsClient method. In tests they are always mocked so they should be ignored by Jacoco